app/controllers/api/v1/product_versions_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    # Controller methods for product_version model
+    class ProductVersionsController < ApplicationController
+      before_action :get_products_id
+      def index
+        @versions = @product.product_versions
+        render json: @versions
+      end
+
+      private
+      def get_products_id
```

Comment:
This method only fetches the product and not the product_id. Also Rubocop suggests that method names should not start with set_* or get_* - https://rubocop.readthedocs.io/en/latest/cops_naming/

Let's rename the method to `def product`. 
What do you think?

Reply:
@aaditi-jain  Done.